### ISSUE TITLE
[BUG] ActiveRecord::RecordNotFound (Couldn't find Project with identifier=undefined):

### ISSUE THREAD

**Issue by dkastl** [gtt-project/redmine_text_blocks#4]
**Describe the bug**

```
F, [2019-08-23T21:05:04.528924 #21015] FATAL -- :   
F, [2019-08-23T21:05:04.529149 #21015] FATAL -- : ActiveRecord::RecordNotFound (Couldn't find Project with identifier=undefined):
F, [2019-08-23T21:05:04.529199 #21015] FATAL -- :   
F, [2019-08-23T21:05:04.529262 #21015] FATAL -- : app/models/project.rb:316:in `find'
plugins/redmine_text_blocks/app/controllers/text_blocks_controller.rb:81:in `find_project_by_project_id'
```

`undefined` cause may be client side.

**Comment by sanak** [comment]
I encounter the same issue, so I will try to solve the issue.

**Comment by sanak** [comment]
This issue only happen when accessible projects is only one and `$("#issue_project_id").val()` returns `undefined`.
https://github.com/gtt-project/redmine_text_blocks/blob/main/assets/javascripts/text_blocks.js#L40
```js
url: "/text_blocks_by_status/"+$("#issue_status_id").val()+"/"+$("#issue_project_id").val(),
```

At this moment, I am thinking to change routes as follows, but more careful investigation will be better to make it sure.
- Before: `/text_blocks_by_status/:status_id/:project_id`
- After: `/text_blocks_by_status/:status_id?project_id=(project_id)&issue_id=(issue_id)`

**Comment by sanak** [comment]
Well, embedding `project_id` seems to be possible, so I keep current API endpoint and fixed this issue on #19.
